app/templating/summary_context.py

```diff
@@ -25,8 +26,18 @@ def build_summary_rendering_context(schema, sections, answer_store, metadata):
     group_ids_on_path = [location.group_id for location in path]
 
     for group in itertools.chain.from_iterable(group_lists):
-        if group['id'] in group_ids_on_path \
-                and schema.group_has_questions(group['id']):
-            groups.extend([Group(group, path, answer_store, metadata, schema).serialize()])
+        if (group['id'] in group_ids_on_path and schema.group_has_questions(group['id'])):
+            no_of_repeats = _number_of_repeats(group, path)
+            repeating_groups = []
+            for instance_idx in range(0, no_of_repeats):
+                summary_group = Group(group, path, answer_store, metadata, schema, instance_idx, schema_context).serialize()
```

Comment:
It will break. I guess the question is whether we want to address this now, or do it when we start implementing deleting repeating groups.
There are going to be a lot of changes once we start removing them, and it probably depends on the removal of `group_instance`. That will presumably entail creating reusable patterns for this kind of stuff.

I'm tempted to leave it for now, but happy to discuss.

Reply:
Fair point. I've raised a card in the backlog about migrating to using `group_instance_id` rather than `group_instance` when we're referring to a group of answers.